costmap_2d/include/costmap_2d/static_layer_with_inflation.h

```diff
@@ -79,6 +79,16 @@ class StaticLayerWithInflation : public CostmapLayer
 
   virtual void onFootprintChanged();
 
+  virtual bool isStaticLayer() {
+    return true;
+  }
+
+  virtual std::shared_ptr<std::vector<double>> getDistancesFromStaticMap() {
```

Comment:
Since we are using c++11 we should probably use the `override` specifier for derived methods.  http://en.cppreference.com/w/cpp/language/override